demos/vlm_npu/README.md

```diff
@@ -252,6 +254,7 @@ Check the [guide of using lm-evaluation-harness](https://github.com/openvinotool
 - models must be exported with INT4 precision and `--sym --ratio 1.0 --group-size -1` params. This is enforced in the export_model.py script when the target_device in NPU.
 - log_probs are not supported
 - finish reason is always set to "stop".
+- only a single response can be returned. Parameter `n` is not supported.
```

Comment:
"python benchmark_serving.py" command does not work:
 ValueError: Initial test run failed - Please make sure benchmark arguments are correctly specified. Error: Bad Request

endopoint for  should be corrected to: --endpoint /v3/chat/completions ?